cmd/kubeadm/app/apis/kubeadm/v1alpha3/doc.go

```diff
@@ -52,41 +76,38 @@ limitations under the License.
 //	  node-cidr-mask-size: 20
 //	schedulerExtraArgs:
 //	  address: "10.100.0.1"
+//  apiServerExtraVolumes:
+//  - name: "some-volume"
+//    hostPath: "/etc/some-path"
+//    mountPath: "/etc/some-pod-path"
+//    writable: true
+//    pathType: File
+//  controllerManagerExtraVolumes:
+//  - name: "some-volume"
+//    hostPath: "/etc/some-path"
+//    mountPath: "/etc/some-pod-path"
+//    writable: true
+//    pathType: File
+//  schedulerExtraVolumes:
+//  - name: "some-volume"
+//    hostPath: "/etc/some-path"
+//    mountPath: "/etc/some-pod-path"
+//    writable: true
+//    pathType: File
 //	apiServerCertSANs:
 //	- "10.100.1.1"
 //	- "ec2-10-100-0-1.compute-1.amazonaws.com"
-//	certificateDirectory: "/etc/kubernetes/pki"
+//	certificatesDir: "/etc/kubernetes/pki"
 //	imageRepository: "k8s.gcr.io"
 //	unifiedControlPlaneImage: "k8s.gcr.io/controlplane:v1.12.0"
-//	auditPolicyConfiguration:
+//	auditPolicy:
```

Comment:
[1]
yep, these names do not match the struct types and are misleading. :\
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go#L116

my vote would be to make them match.